Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.cpp

```diff
@@ -1551,6 +1554,29 @@ int CLuaElementDefs::IsElementFrozen(lua_State* luaVM)
     return 1;
 }
 
+int CLuaElementDefs::IsElementOnGround(lua_State* luaVM)
+{
+    // Verify the argument
+    CClientEntity*   pEntity = NULL;
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadUserData(pEntity);
+
+    if (!argStream.HasErrors())
+    {
+        bool bOnGround;
+        if (CStaticFunctionDefinitions::IsElementOnGround(*pEntity, bOnGround))
+        {
+            lua_pushboolean(luaVM, bOnGround);
+            return 1;
+        }
```

Comment:
first of all, use new parser, second: if "isElementOnGround" return false -> throw ""error, element type not supported""